factcast-itests/factcast-itests-factus/src/test/java/org/factcast/itests/factus/client/DynamoITest.java

```diff
@@ -300,7 +300,7 @@ public WriterToken acquireWriteToken(@NonNull Duration maxWait) {
     }
 
     @Override
-    protected void apply(UserCreated created) {
+    public void apply(UserCreated created) {
```

Comment:
handler methods should not be public, we even print a warning there.
Why is it necessary?

Reply:
changed it

Reply:
AH! because you pulled them in the interface, thus making them implicitely public.
Sorry, did not get that.